group/group.go

```diff
@@ -35,6 +35,8 @@ type Element interface {
 	Copy() Element
 	IsIdentity() bool
 	IsEqual(Element) bool
+	CMov(int, Element) Element
```

Comment:
Document.

Reply:
In particular: what happens of the `int` isn't 0 or 1?

Reply:
All documentation was added in #355